server/src/routers/coordinatesRouter.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const router = new express.Router()
+
+const Coordinates = require('../models/coordinates')
+
+const coordinatesValidator = require('../middlewares/coordinatesHandlers')
+
+//The distance should be in meters from which we need to find the parking ads
+router.get('/search', coordinatesValidator, async (req, res, next) => {
+    const longitude = req.body.longitude
+    const latitude = req.body.latitude
+
+    try {
+        let parkingAds = await Coordinates.find({
+                location: {
+                    $nearSphere: {
+                        $geometry: {
+                            type: "Point",
+                            coordinates: [longitude, latitude],
+                        },
+                        $minDistance: 0,
+                        $maxDistance: req.body.distance
+                    }
+                }
```

Comment:
Just out of curiosity, can you direct me to the documentation for this search of locations?

Reply:
https://mongoosejs.com/docs/geojson.html